.drone.yml

```diff
@@ -50,6 +50,7 @@ steps:
     pull: always
     environment:
       PLATFORM: linux/amd64,linux/arm64
```

Comment:
it's too late, but we should probably remove this to simplify things: we don't want to run tests on arm64 (using test built for arm64)